crates/sui-types/src/executable_transaction.rs

```diff
@@ -63,3 +66,33 @@ impl VerifiedExecutableTransaction {
         }
     }
 }
+
```

Comment:
is this a good home for those functions?
@lxfind @tnowacki @mystenmark @tzakian @amnn @bmwill

Reply:
Probably not. I think sui-types/execution.rs would be a better place.

Reply:
will do, thanks!